ui/src/components/getRun/NameValueTable.jsx

```diff
@@ -0,0 +1,60 @@
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableRow from '@material-ui/core/TableRow';
+import { makeStyles } from '@material-ui/core/styles';
+
+import LoadingShimmer from './LoadingShimmer';
+
+const useStyles = makeStyles(_ => ({
+  table: {
+    '& > .MuiTableCell-root': {
+      borderBottom: 'none',
+    },
+  },
+  row: {
+    '& > td': {
+      borderBottom: 'none',
+      fontSize: '18px',
+      lineHeight: '24px',
+    },
+    '& > td:first-child': {
+      color: '#707070',
+    },
+  },
+}));
+
+export const makeRow = (name, value) => ({
+  name,
+  value,
+});
+
+export const NameValueTable = ({ rows, rowsToLoad }) => {
```

Comment:
Before seeing the body I read `rowsToLoad` as the subset of `rows` to load. Know how to use the component/function shouldn't require reading its implementation.
```suggestion
export const NameValueTable = ({ rows, numToLoad }) => {
```
and/or typing it as `number`.

Reply:
Done